docs/developer/discounts/vouchers.mdx

```diff
@@ -5,6 +5,11 @@ sidebar_label: Vouchers
 
 ## Introduction
 
+Vouchers are a way to apply discounts to orders. They can be used to reduce the cost of a checkout.
+Vouchers can be applied to the entire order or to specific products. They can also be limited to a minimum quantity of products in the checkout.
```

Comment:
Maybe we should also mention here that the vouchers can be mixed with promotions?